Tests/SpelledPitchTests/PitchSpellerTests.swift

```diff
@@ -352,11 +359,33 @@ class PitchSpellerTests: XCTestCase {
 
         let pitches: [Int: Pitch] = [1: 61, 5: 65]
         let pitchSpeller = PitchSpeller(pitches: pitches, parsimonyPivot: .d)
-
-        XCTAssertEqual(pitchSpeller.flowNetwork.nodes,  expected.nodes)
-        XCTAssertEqual(pitchSpeller.flowNetwork.edges,  expected.edges)
-        XCTAssertEqual(pitchSpeller.flowNetwork.sink,   expected.sink)
-        XCTAssertEqual(pitchSpeller.flowNetwork.source, expected.source)
         XCTAssertEqual(pitchSpeller.flowNetwork, expected)
     }
 }
+
+
+extension Cross: CustomStringConvertible {
+    public var description: String {
+        return "(\(a),\(b))"
+    }
+}
+
+extension OrderedPair: CustomStringConvertible {
+    public var description: String {
+        return "(\(a),\(b))"
+    }
+}
+
+extension FlowNode: CustomStringConvertible {
+
+    public var description: String {
+        switch self {
+        case .source:
+            return "source"
+        case .sink:
+            return "sink"
+        case .internal(let index):
+            return "\(index)"
+        }
+    }
+}
```

Comment:
These are great! Let's actually merge some of these changes in... ?

Reply:
Just want the `CustomStringConvertible` bits?

Reply:
Yeah, I have a feeling those will be very handy when tests fail

Reply:
Do they show up in tests?

Reply:
When they fail, yes. Without the `CustomStringConvertible` its like 30 pages long 😭 .

Reply:
K, I'll do another PR with them in the right place.

Reply:
And yes, I have read through those failures in times of bemusement!

Reply:
I usually end up copying into `vim` and diving into the world of selective deletion, but this is the right way to go about things (note to self)